packages/aws/data_stream/securityhub_findings/elasticsearch/ingest_pipeline/default.yml

```
@@ -1405,10 +1520,328 @@ processors:
       field: json.Remediation.Recommendation.Url
       target_field: aws.securityhub_findings.remediation.recommendation.url
       ignore_missing: true
+  - set:
+      field: rule.reference
+      tag: set_rule_reference
+      copy_from: aws.securityhub_findings.remediation.recommendation.url
+      ignore_empty_value: true
+  - set:
+      field: rule.references
+      tag: set_rule_references
+      copy_from: aws.securityhub_findings.remediation.recommendation.url
+      ignore_empty_value: true
+  - set:
+      field: rule.remediation
+      tag: set_rule_remediation
+      value: "{{{aws.securityhub_findings.remediation.recommendation.text}}} \r\n {{{aws.securityhub_findings.remediation.recommendation.url}}}"
```

```suggestion
      value: "{{{aws.securityhub_findings.remediation.recommendation.text}}}\r\n{{{aws.securityhub_findings.remediation.recommendation.url}}}"
```
Otherwise there is a trailing space on the first line and a leading space on the second.

---

updated as suggested.